diff -urNp -x '*.orig' librepo-1.8.1.org/librepo/CMakeLists.txt librepo-1.8.1/librepo/CMakeLists.txt
--- librepo-1.8.1.org/librepo/CMakeLists.txt	2017-09-01 17:45:17.000000000 +0200
+++ librepo-1.8.1/librepo/CMakeLists.txt	2021-03-05 22:44:35.021105057 +0100
@@ -52,6 +52,7 @@ TARGET_LINK_LIBRARIES(librepo
                         ${LIBCRYPTO_LIBRARIES}
                         ${GPGME_VANILLA_LIBRARIES}
                         ${GLIB2_LIBRARIES}
+			crypto
                      )
 SET_TARGET_PROPERTIES(librepo PROPERTIES OUTPUT_NAME "repo")
 SET_TARGET_PROPERTIES(librepo PROPERTIES SOVERSION 0)
