diff -dur librepo-librepo-1.7.13.orig/doc/python/CMakeLists.txt librepo-librepo-1.7.13/doc/python/CMakeLists.txt
--- librepo-librepo-1.7.13.orig/doc/python/CMakeLists.txt	2015-01-23 15:43:51.000000000 +0100
+++ librepo-librepo-1.7.13/doc/python/CMakeLists.txt	2015-11-30 18:48:12.275656366 +0100
@@ -1,5 +1,5 @@
 ADD_CUSTOM_TARGET (doc-python
-    PYTHONPATH=${CMAKE_BINARY_DIR}/librepo/python/python${PYTHON_DESIRED} sphinx-build -E -b html
+	PYTHONPATH=${CMAKE_BINARY_DIR}/librepo/python/python${PYTHON_DESIRED} ${SPHINX_EXECUTABLE} -E -b html
                   ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}
                   COMMENT "Building Python API documentation with Sphinx")
 
